lib/pages/chat/chat_pinned_events/pinned_messages.dart

```diff
@@ -189,6 +190,27 @@ class PinnedMessagesController extends State<PinnedMessages>
     }
   }
 
+  void onLongPressMessage(BuildContext context, Event event) {
+    if (PinnedMessagesStyle.responsiveUtils.isMobile(context)) {
+      _showMessageBottomSheet(event);
+    } else {
+      onSelectMessage(event);
```

Comment:
change the name to `selectMessage`

Reply:
though should delete it cause no need for long press on web